generator/README.md

```diff
@@ -104,7 +104,7 @@ Put the extracted mibs in a location NetSNMP can read them from. `$HOME/.snmp/mi
 * UCD-SNMP-MIB (Net-SNMP): http://www.net-snmp.org/docs/mibs/UCD-SNMP-MIB.txt
 * Ubiquiti Networks: http://dl.ubnt-ut.com/snmp/UBNT-MIB
                      http://dl.ubnt-ut.com/snmp/UBNT-UniFi-MIB
-                     https://community.ubnt.com/ubnt/attachments/ubnt/airMAX_FAQ/84/3/ubnt-mib.04.06.2017.zip
+                     https://www.ubnt.com/downloads/firmwares/airos-ubnt-mib/ubnt-mib.zip
```

Comment:
I just looked at my test more closely, it looks like this URL is 301'd to `https://dl.ubnt.com/firmwares/airos-ubnt-mib/ubnt-mib.zip`.  I would suggest using this URL.

Reply:
LGTM

Reply:
Updated to pull from that final redirect